lib/asciidoctor/converter/semantic_html5.rb

```diff
@@ -67,6 +67,10 @@ def convert_paragraph node
     end
   end
 
+  def convert_thematic_break node
+    "<hr />"
```

Comment:
I think we should use `<hr>` (at least for now).
In my opinion, we should open an issue to discuss if we want to support XHTML in the new converter or not.

Reply:
fixed with 0861630